doc/tutorial.rst

```diff
@@ -757,14 +766,14 @@ Our top-level code looks like this:
     def print_error(filename, sloc_range, message):
         line = sloc_range.start.line
         column = sloc_range.start.column
-        print >> sys.stderr, 'In {}, line {}:'.format(filename, line)
+        print ('In {}, line {}:'.format(filename, line), sys.stderr)
```

Comment:
shouldn't it be `print("...", file=sys.stderr)` instead of `print("...", sys.stderr)` to avoid the `<_io.TextIOWrapper name='<stderr>' mode='w' encoding='utf-8'>` stuff, as we saw together? (same below)

Reply:
correct, I forgot to fix these ones!